netbox/extras/forms/model_forms.py

```diff
@@ -78,6 +84,20 @@ def __init__(self, *args, **kwargs):
             self.fields['type'].disabled = True
 
 
+class CustomFieldChoiceSetForm(BootstrapMixin, forms.ModelForm):
+    extra_choices = forms.CharField(
+        widget=ArrayWidget(),
+        help_text=_('Enter one choice per line.')
+    )
+
+    class Meta:
+        model = CustomFieldChoiceSet
+        fields = ('name', 'description', 'extra_choices', 'order_alphabetically')
+
+    def clean_extra_choices(self):
+        return self.cleaned_data['extra_choices'].split('\n')
```

Comment:
This does not account for Windows users who get CRLF instead of just LF.

```suggestion
        return self.cleaned_data['extra_choices'].replace('\r\n','\n').split('\n')
```

Reply:
This should use `splitlines()` to catch both cases.